deploy/Makefile

```diff
@@ -1,8 +1,6 @@
 STACK ?= validator
 CLOUD ?= azure
 
-include ./cmd/deploy/assets/Makefile
```

Comment:
Why? `make -C deploy` is better than `make -C deploy/cmd/deploy/assets`

Reply:
got relative path problems due to `include` another makefile, now manually proxy those commands at `deploy/Makefile`